mail/views.py

```diff
@@ -84,11 +85,17 @@ class AutomaticEmailView(ListAPIView):
     )
     permission_classes = (permissions.IsAuthenticated, UserCanMessageLearnersPermission, )
     serializer_class = AutomaticEmailSerializer
+    lookup_field = "id"
+    lookup_url_kwarg = "email_id"
 
     def get_queryset(self):
         """Get the queryset which should be serialized"""
         return AutomaticEmail.objects.filter(staff_user=self.request.user)
 
+    def patch(self, request, *args, **kwargs):
```

Comment:
I believe without it I got an error but I don't remember exactly, I can remove it and see what happens